commands/scan/buildscan.go

```diff
@@ -186,7 +191,21 @@ func (bsc *BuildScanCommand) CommandName() string {
 	return "xr_build_scan"
 }
 
-func trimBuildScanResultUrl(fullUrl string) (string, string, error) {
+// There are two cases. when serverDetails.Url is configured and when serverDetails.XrayUrl and serverDetails.ArtifactoryUrl are configured
+// The function will return the Url if configured and will trim xray if serverDetails.Url is not configured
+func getActualUrl(serverDetails config.ServerDetails) (string, error) {
+	url := serverDetails.Url
```

Comment:
```suggestion
	url := trimUrl(serverDetails.Url)
```
should we trim `serverDetails.Url` as well?